app/models/storage.rb

```diff
@@ -68,6 +68,8 @@ class Storage < ApplicationRecord
   virtual_column :total_unmanaged_vms,            :type => :integer  # uses is handled via class method that aggregates
   virtual_column :count_of_vmdk_disk_files,       :type => :integer
 
+  delegate :queue_name_for_ems_operations, :to => :ext_management_system, :allow_nil => true
+
```

Comment:
Note: this uses `MiqQueue.submit_job` [link](https://github.com/ManageIQ/manageiq/blob/9bdb3a154b9c6d5ede5261614c0c512eb9f792ff/app/models/storage.rb#L202-L203) with the service ems_operations service, so MiqQueue.submit_job will automatically set the queue_name to `resource.queue_name_for_ems_operations` if that is present so there's no need to set a queue_name explicitly